packages/carbon-web-components/src/components/tag/tag.ts

```diff
@@ -41,10 +108,36 @@ class BXTag extends LitElement {
   type = TAG_TYPE.GRAY;
 
   render() {
-    return html` <slot></slot> `;
+    const { disabled, filter } = this;
+    return html`
+      <slot></slot>
+
+      ${filter
+        ? html`
+            <button class="${prefix}--tag__close-icon" ?disabled=${disabled}>
+              ${Close16({ 'aria-label': this.title })}
```

Comment:
should also be able to render their own icon: 
<img width="814" alt="Screen Shot 2023-03-20 at 12 55 55 PM" src="https://user-images.githubusercontent.com/20210594/226438960-6a54bb9e-237a-4f5a-967d-4a1338c91d20.png">